api/app/controllers/mno_enterprise/jpi/v1/app_answers_controller.rb

```diff
@@ -0,0 +1,22 @@
+module MnoEnterprise
+  class Jpi::V1::AppAnswersController < Jpi::V1::AppReviewsController
+
+    private
+
+    def scope_app_reviews
```

Comment:
I would have preferred that the app_reviews takes the app_reviews as parameter, so that it does not rely on calling the member.